cdap-docs/admin-manual/source/installation/installation-note.txt

```diff
@@ -0,0 +1,18 @@
+.. topic:: Different ways to install CDAP
+
+  There are different ways to install CDAP, depending if it is a distributed or standalone system:
+
+  - For **CDH (Cloudera)**, you can follow :ref:`these instructions <cloudera-configuring>` using Cloudera Manager.
```

Comment:
this implies you cannot install on CDH without cloudera manager.  "For a CDH cluster managed by Cloudera Manager..."